fedora.txt

```diff
@@ -9,3 +9,17 @@ sudo python3 -m unittest tunirtests.cloudservice.TestServiceManipulation -v
 @@ sudo reboot
 SLEEP 30
 sudo python3 -m unittest tunirtests.cloudservice.TestServiceAfter -v
+
+sudo python3 -m unittest tunirtests.atomictests.TestDockerInstalled -v
+sudo python3 -m unittest tunirtests.atomictests.TestDockerStorageSetup -v
```

Comment:
Maybe add `TestAtomicFirstBootRun()` where we run `docker run --rm busybox true && echo "PASS"` on first boot.